src/lambdaisland/glogc.cljc

```diff
@@ -51,9 +51,13 @@
        (target :clj  (#'pedestal/log-expr &form :error keyvals)
                :cljs (#'glogi/log-expr &form :error keyvals)))
 
-     (defmacro spy [expr]
-       (target :clj `(pedestal/spy ~expr)
-               :cljs `(glogi/spy ~expr)))
+     (defmacro spy
+       ([expr]
+        (target :clj `(pedestal/spy ~expr)
+                :cljs `(glogi/spy ~expr)))
+       ([expr & exprs]
+        (target :clj `(pedestal/spy ~expr)
```

Comment:
I guess in the case of clj/pedestal we can `(pedestal/spy [~expr ~@exprs])`

Reply:
Done!